app/generate/[slug]/page.tsx

```diff
@@ -0,0 +1,9 @@
+export default function Page({ params }: { params: {slug: string } }) {
+  
+    return (
+      <>
+        <h1>Hi there !</h1>
+        <p>You will generate a video with the title {params.slug}</p>
+      </>
+    );
+  }
```

Comment:
You used this page to test the next routing slugs but couldn't we just keep one page knowing that we won't use the others ?

Reply:
Sure i've removed it and we can keep the page in /generate 👍